src/microvis/backend/vispy/_canvas.py

```diff
@@ -69,3 +69,8 @@ def _vis_render(
             region=region, size=size, bgcolor=bgcolor, crop=crop, alpha=alpha
         )
         return cast("np.ndarray", data)
+
+    def _vis_get_ipython_mimebundle(
+        self, *args: Any, **kwargs: Any
+    ) -> dict | tuple[dict, dict]:
+        return self._vis_get_native()._repr_mimebundle_(*args, **kwargs)  # type: ignore
```

Comment:
type ignore here, was complaining about returning Any in a typed context - wasn't sure if it was solvable so left